packages/components/src/__future__/Tag/_docs/Tag.mdx

```diff
@@ -0,0 +1,33 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as TagStories from "./Tag.stories"
+
+<Meta of={TagStories} />
+
+# Avatar
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/__future__/Tag"
+  figma="https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%F0%9F%96%BC%EF%B8%8F-Component-Gallery?type=design&node-id=9-37843&mode=design&t=v8aGjXZU6apybb6y-0"
+  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3293283155/Tag"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement="import { Tag } from `@kaizen/components/future`"
+/>
+
+## Overview
+
+Tags helps users quickly recognize important information about items that organize and categorize them.
+
+<Canvas of={TagStories.Playground} />
+<Controls of={TagStories.Playground} />
+
+## API
+
+Coming Soon
```

Comment:
This is a good time to fill in the api section! Create stories for the props and give them some descriptions :)
You can take a look at the Filter(Bar) ones for inspiration - it's likely much simpler for Tag.

Reply:
Or `Button` 😄